src/com/owncloud/android/datamodel/OCFile.java

```diff
@@ -60,6 +60,47 @@ public OCFile createFromParcel(Parcel source) {
     public static final String PATH_SEPARATOR = "/";
     public static final String ROOT_PATH = PATH_SEPARATOR;
 
+    public enum AvailableOfflineStatus {
+
+        /**
+         * File is not available offline
+         */
+        NOT_AVAILABLE_OFFLINE(0),
+
+        /**
+         * File is available offline
+         */
+        AVAILABLE_OFFLINE(1),
+
+        /**
+         * File belongs to an available offline folder
+         */
+        AVAILABLE_OFFLINE_PARENT(2);
+
+        private final int value;
+
+        AvailableOfflineStatus(int value) {
+            this.value = value;
+        }
+
+        public int getValue() {
+            return value;
+        }
+
+        public static AvailableOfflineStatus fromValue(int value) {
```

Comment:
Sure, though is it a big win? Keeping an ordinal relationship between the elements is not important for our case, and the attribute is not too heavy. I'd rather to have control about the values than about the order.

Is there any other benefit I'm losing?